problem12.rb

```diff
@@ -9,11 +9,10 @@ def NumberOfDivisors(number)
     return nod
 end
  
- 
 def problem
   number = 0
   i = 1
```

Comment:
```suggestion
  number, i = 0, 1
```
https://stackoverflow.com/questions/2895957/parallel-assignment-operator-in-ruby